src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/ChooseEntityTypeType.php

```diff
@@ -27,19 +27,25 @@
 
 class ChooseEntityTypeType extends AbstractType
 {
+    private $formCount = 0;
```

Comment:
Why is this needed?

Reply:
That enables us to render two forms of the same type on a page. The counter value is used to name the form and thus making it unique.

Reply:
Yes I where it's for but I don't understand why it's needed.

Reply:
The protocol selection popup is rendered twice on the page for both test and production entities. And previously they where named the same. This caused problems when handling the form. This was also evident in the front-end form behaviour. For example, clicking on a form label element would focus the corresponding input element on the other, out of sight, form.